gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -400,13 +403,23 @@ public String getEndpoint() {
     return endpoint;
   }
 
+  @ObsoleteApi("Use getKeepAliveTimeDuration() instead")
+  public org.threeten.bp.Duration getKeepAliveTime() {
+    return toThreetenDuration(getKeepAliveTimeDuration());
+  }
+
   /** The time without read activity before sending a keepalive ping. */
-  public Duration getKeepAliveTime() {
+  public java.time.Duration getKeepAliveTimeDuration() {
```

Comment:
This looks like a breaking change. Have we released with `Duration getKeepAliveTime()`?

Reply:
@burkedavison we keep the method signature in 
https://github.com/googleapis/sdk-platform-java/blob/7774eb040162810d75263b2835c46175f81959ae/gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java#L440-L443

It is not a breaking change in terms of signature, but the implementation of the `threeten` version has changed to become a wrapper of the `java.time` version.